records_mover/db/postgres/copy_options/text.py

```diff
@@ -1,16 +1,20 @@
 from records_mover.utils import quiet_remove
 from records_mover.records.hints import cant_handle_hint
 from records_mover.records.types import RecordsHints
-from typing import Set, Optional, Tuple
-from .date_input_style import DateInputStyle
+from typing import Set, Optional, Tuple, Union, Literal
 from .types import PostgresCopyOptions
+from .mode import CopyOptionsMode
 from .common import postgres_copy_options_common
 
 
+# TODO: Audit this file
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker